packages/manager-components/src/components/navigation/menus/action/action.component.tsx

```diff
@@ -28,15 +30,23 @@ export interface ActionMenuItem {
   target?: OdsHTMLAnchorElementTarget;
   onClick?: () => void;
   label: ReactI18NextChild | Iterable<ReactI18NextChild>;
+  action?: string;
```

Comment:
**nitpick:** Considering if we implement any code-annotations, it would be nice to rename it to iamAction to easily identify the actions only related to iam.

Reply:
Ok I rename it 👍